R/clustering.R

```diff
@@ -13,6 +13,9 @@ NULL
 #' @importFrom pbapply pblapply
 #' @importFrom future.apply future_lapply
 #' @importFrom future nbrOfWorkers
+#' @importFrom igraph graph_from_adjacency_matrix
+#' @importFrom methods as
+#' @importClassesFrom Matrix dgCMatrix
```

Comment:
Why are we importing the dgCMatrix class? This should only be necessary if we're creating a subclass, which doesn't seem to be happening.

Reply:
Ah, sorry I misunderstood if importing class/methods was needed.